README.md

```diff
@@ -1,5 +1,53 @@
-# Heroku Cloud Native PHP Buildpack
+# Heroku Cloud Native Buildpack: PHP
 
-[![CI](https://github.com/heroku/buildpacks-php/actions/workflows/ci.yml/badge.svg)](https://github.com/heroku/buildpacks-php/actions/workflows/ci.yml)
+[![Cloud Native Buildpacks Registry: heroku/php][registry-badge]][registry-url]
+[![CI on Github Actions: heroku/php][ci-badge]][ci-url]
 
-Heroku's official [Cloud Native Buildpack](https://buildpacks.io) for the PHP ecosystem.
+![Heroku Cloud Native Buildpack: heroku/php][cnb-banner]
```

Comment:
(Optional) One downside to this banner image is that it says "official Heroku buildpack", which isn't quite accurate at this point, since (a) these can't be used on Heroku yet, (b) the classic buildpack is technically the "official" (ie default) one. Unless we have access to the original image assets this probably isn't something we can easily fix - so not sure what the alternatives are here (some other random image? omit the image?)

Reply:
Yes, I'd rather have an official Heroku Cloud Native Buildpack variant of this, but I don't think we'll have one soon. I think yeah, the two options are include this, or don't. We discussed this briefly [here](https://salesforce-internal.slack.com/archives/C06L74ZHGRX/p1709159592853549). I personally like keeping it to give an official feel.